content/docs/04-clusters/02-data-center/01-maas/15-create-manage-maas-clusters.md

```diff
@@ -0,0 +1,155 @@
+| Availability Zones | These are multiple, isolated locations within a Region. When you choose one or more availability zones, Palette provides fault tolerance to guard against hardware failures and network failures by provisioning nodes across availability zones if multiple zones are selected. |
+| Disk Size | Specify the required storage size. |
+
+
+12. You can configure the following cluster management features now if needed, or you can do it later:
+
+- Manage Machines
+- Scan Policies
+- Backup Policies
+- Review settings and deploy the cluster. 
+
+13. Review settings and deploy the cluster. 
+
+
+# Validation
+
+You can validate your cluster is available by reviewing the cluster details page. Navigate to the left **Main Menu** and click **Clusters**. The **Clusters** page lists all available clusters Palette manages. Select the cluster to review its details page. Ensure the **Cluster Status** field contains the value **Running**.
+
+
+
+# Delete a MAAS Cluster
+
+When you delete a MAAS cluster, all virtual machines and associated storage disks that were created for the cluster are removed. 
+
+Follow these steps to delete a MAAS cluster.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and click **Clusters**. 
+
+
+3. Select the cluster you want to delete.
+
+
+4. Click the **Settings drop-down Menu**, and choose **Delete**.
+
+The cluster status is updated to Deleting while cluster resources are being deleted. When all resources are successfully deleted, the cluster status is updated to Deleted and the cluster is removed from the list. The delete operation returns the edge hosts to the **Ready** state. All the artifacts related to the Kubernetes distribution are removed.
+
+
+## Force Delete a Cluster
+
+If a cluster is stuck in the Deletion state for a minimum of 15 minutes it becomes eligible for force deletion. You can force delete a cluster from the tenant and project admin scope. To force delete a cluster follow the same steps outlined in [Delete a MAAS Cluster](/clusters/data-center/maas/create-manage-maas-clusters#deleteamaascluster). However, after 15 minutes, a **Force Delete Cluster** option is available in the **Settings drop-down menu**. The drop-down menu will provide you with an estimated time left before the force deletion becomes available.
+
+<br />
+
+<WarningBox>
+A force delete can result in Palette-provisioned resources to be missed in the removal process. Verify there are no remaining resources. Failure to remove provisioned resources can result in unexpected costs.
+</WarningBox>
+
+# Next Steps
+
+Now that you’ve deployed a MAAS cluster, you can start developing and deploying applications to your cluster. We recommend you review the Day-2 responsibilities and become familiar with the cluster management tasks. Check out the [Manage Clusters](/clusters/cluster-management) documentation to learn more about Day-2 responsibilities.
```

Comment:
not sure what Day-2 "responsibilities" means. Do we mean "operations" perhaps?

Reply:
@antongisli yes, this should be "operations". This was from the original doc. Thanks for catching it.